docs/source/testing/trip_test.rst

```diff
@@ -0,0 +1,119 @@
+Trip Tests
+==============
+
+Summary
+-------
+The primary purpose of the trip tests is to identify if proposed changes to the Isca master result in it giving different results for a range of test cases. 
+
+The trip test program takes two commit IDs as input, and compares the results of Isca's test cases using the two versions of Isca that are defined by the commit IDs. A comparison is then made between the output from each of the test cases. The output of the program shows which test cases produced bit-wise identical results and which did not.
+
+
+Why do we ask you to run the trip tests?
+-----------------------------------------
+
+We are very keen for people to contribute their modifications and enhancements back into Isca's master - for instructions on how to contribute, see the :doc:`../contributing` . As part of this process, before we accept your modifications, we require you to run the trip tests. By running these tests, we can check whether your modifications changes the results of any of the test cases.
+
+We want to check this for three principal reasons:
+
+    1. If you add a new feature, we want to make sure it is turned off by default. This is so that an existing Isca user who doesn't know about the feature isn't forced to use it when they upgrade to the newest Isca version. This prevents confusion where two versions of Isca give different answers for the same experimental configuration.
+    2. If your modification contains a bug fix, we want to know which test cases are affected. We also want to know which pull request resulted in the bug fix.
+    3. You may think your modifications won't have changed Isca's results, but we want to make sure that's the case. The trip tests are very good at highlighting unexpected consequences of your changes, and help us keep to a high-standard of reproducibility.
+
+
+How do I run the trip tests?
+---------------------------------
+The first thing to do before you run the trip tests is to decide on which two commit IDs you would like to compare, and then obtain their commit IDs. When you submit a pull request, we ask you to use the current HEAD commit of Isca's master branch as one of the commits, and the HEAD of your pull-requested branch as the other. If you are using the trip tests as a testing tool, then you can choose any two commits you like.
+
+To run the trip tests, navigate to ``${GFDL_BASE}/exp/test_cases/trip_test/``. From this directory the trip tests are run using the command-line interface. The simplest example call is
+::
+    ./trip_test_command_line 155661f ec29bf3
+This would run all of the test cases, comparing the results using commit ID ``155661f`` with commit ID ``ec29bf3``.
```

Comment:
I think you need to be even more explicit for people who are not experienced with git. Suggest you add a line here to say: these commit IDs are for illustration purposes, please use two commit IDs relevant for your testing (i.e. the currentHEAD of the Isca master and the HEAD of your branch).

Reply:
Good suggestion - thanks.